tests/test_cases/test_discovery/test_vhdl_block.py

```diff
@@ -0,0 +1,44 @@
+''' Copyright (c) 2015 Potential Ventures Ltd
```

Comment:
Could you please make this a `#` comment? We moved to those for the license header to help the autodoc generation, and I don't want to add new instances of the old style.